Documentation/Examples/Index.rst

```diff
@@ -12,29 +12,56 @@ Extension styleguide
 ====================
 
 Many of the examples are part of the TYPO3 extension :file:`styleguide`. This
-extension can be installed via composer:
+extension offers neatly numbered examples for any field type. Furthermore there
+are examples with different properties set to different values.
 
-.. code-block:: console
-
-   composer require --dev typo3/cms-styleguide
+Read here about how to :ref:`install and use the styleguide extension
+<tca_examples_styleguide_installations>`.
 
-It can also be downloaded from `GitHub <https://github.com/TYPO3/styleguide>`__.
-
-.. todo:
-   Explain how to use the extension styleguide.
-   See https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-TCA/issues/297
 
 .. _tca_examples_extension_examples:
 
 Extension examples
 ==================
 
-Some examples can be found in the TYPO3 extension :file:`examples`. It can
-be installed via composer:
+Some examples can also be found in the TYPO3 extension :file:`examples`. This
+extension contains special configurations that have been used neither in the
+extension :file:`stylguide` not in the TYPO3 Core.
```

Comment:
```suggestion
extension contains special configurations that have not been used in the
extension :file:`stylguide` nor in the TYPO3 Core.
```

Reply:
Or "...that haven't been used in either the extension :file:`styleguide` or in the TYPO3 Core."